src/slurm_plugin/clustermgtd.py

```diff
@@ -150,6 +150,7 @@ class ClustermgtdConfig:
         "terminate_drain_nodes": True,
         "terminate_down_nodes": True,
         "orphaned_instance_timeout": 300,
+        "ec2_backing_instance_max_count": 2,
```

Comment:
I feel the naming of this var doesn't communicate it's purpose.
If I don't know the behind the scene on how this value it's used, the variable name should tell me that this is the limit we are allowing an instance to be missing from EC2.

Reply:
I agree and I was struggling a bit with this name.  I thought it was getting too long lol

something like `ec2_instance_missing_max_count`?